onward/app/controllers/MostPopularController.scala

```diff
@@ -31,10 +31,9 @@ object MostPopularController extends Controller with Logging with ExecutionConte
     val sectionPopular: Future[List[MostPopular]] = if (path.nonEmpty) lookup(edition, path).map(_.toList) else Future(Nil)
 
     sectionPopular.map { sectionPopular =>
-      val sectionFirst = sectionPopular ++ globalPopular
-      val globalFirst = globalPopular.toList ++ sectionPopular
-
-      val mostPopular = if ( path.contains("global-development") ) sectionFirst else globalFirst
+      lazy val sectionFirst = sectionPopular ++ globalPopular
+      lazy val globalFirst = globalPopular.toList ++ sectionPopular
+      lazy val mostPopular = if ( path.equals("global-development") ) sectionFirst else globalFirst
```

Comment:
I think == is more scala and .equals is more java.
Google probably has many opinions, but here's one http://stackoverflow.com/questions/7681161/whats-the-difference-between-and-equals-in-scala